src/getUserConfigFile.test.js

```diff
@@ -0,0 +1,50 @@
+import fs from 'fs-extra';
+import { getUserConfigFile } from './config';
+
+jest.mock('fs-extra', () => ({
+  existsSync: jest.fn(),
+}));
+
+describe('getUserConfigFile', () => {
+  const mockedCustomConfig = {
+    ROOT_DIR: 'dir',
+    CYPRESS_SCREENSHOT_OPTIONS: { disableTimersAndAnimations: false }
+  };
+
+  const mockedDefaultConfig = {
+    ROOT_DIR: '',
+    CYPRESS_SCREENSHOT_OPTIONS: { disableTimersAndAnimations: false }
+  };
+
+  beforeEach(() => {
+    jest.resetModules();
+  });
+
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it('should return config from .cjs file if config file exists', () => {
+    fs.existsSync.mockReturnValue(true);
+    jest.doMock(`${process.cwd()}/cypress-image-diff.config.cjs`, () => mockedCustomConfig, { virtual: true });
+
+    const config = getUserConfigFile();
+    expect(config).toEqual(mockedCustomConfig);
+  });
+
+  it('should return default config if custom config does not exist', () => {
+    fs.existsSync.mockReturnValue(false);
+    const config = getUserConfigFile();
+    expect(config).toEqual(mockedDefaultConfig);
+  });
+
```

Comment:
I think it would be if we could have a test case where users have partial config file. We should be able to pick up the custom config as well as the default ones. The rest looks great to me.

Reply:
Great idea! Will add soon

Reply:
@kien-ht I haven't found a way to test `userConfig` as it was a constant before even importing it to the test. It might be just my lack of knowledge, but didn't manage to test it this way.

- I changed `userConfig` to a `getUserConfig` function, although I am not sure if that is even a good idea. Updated in the affected places as well.

Let me know if I should undo the changes (feel free to push back) and skip `userConfig`/`getUserConfig` tests. Not sure if using it as a function would affect any behaviour :cry: